config.yml.example

```diff
@@ -182,7 +182,9 @@ logging:
   maxFileSizeMB: 15
   compressOnRotation: false
   
-checkForUpdatesAtBoot: true
+checkForUpdatesAtBoot: false
+checkForUpdatesInterval: 432000000 #  1000 * 3600 * 24 * 5 // Check every 5 days
```

Comment:
Would it be easier or more readable to do this in days?

Reply:
The BGPAlerter source parameter is milliseconds, it was hardcoded. I'm just pull the property out into a config file